### DISCUSSION TITLE
Want more achievements badges.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#81485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi github i guess adding more achievements to profiles is very joyful. So i just request some more achievements for profiles of upcoming programmers

**Comment by gabrielgxrcia** [comment]
I also think this idea is nice 😊

**Reply by ghost** [reply]
Thank you, I guess there are people who thinks like me in this scenario

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your  ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Ayzzzzzzzzzzz** [comment]
yea would be nice
